GoogleForCreators/web-stories-wp
# Fix auto-setting rotated background

## Context
Previously, rotation was not properly considered when auto-setting background, causing setting background even if the media was only partially covering the background area.


## Summary
This PR re-uses the logic implemented in #11163 to calculate properly if the element is truly covering the background area fully before setting it as a background.


## Relevant Technical Choices



## To-do



## User-facing changes
Background is not set when there's a part that's not covered:

![autobg](https://user-images.githubusercontent.com/3294597/178289599-ba8e3efe-61c1-4cc6-989b-c46c72551f6d.gif)



## Testing Instructions






This PR can be tested by following these steps:

1. Add a media and rotate it
2. Now resize it so that it would almost cover the background area but not fully, e.g. leave one corner uncovered
3. Verify the media is not auto-set as background
4. Now resize it to cover fully and verify it's now auto-set.


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #10519
